test/tests/auxkernels/bounds/constant_bounds_elem.i

```diff
@@ -0,0 +1,142 @@
+[Mesh]
+  type = GeneratedMesh
+
+  dim = 1
+
+  xmin = 0
+  xmax = 1
+
+  nx = 10
+[]
+
+[Variables]
+  [u]
+    order = CONSTANT
+    family = MONOMIAL
+  []
+
+  [v]
+    order = CONSTANT
+    family = MONOMIAL
+  []
+[]
+
+[AuxVariables]
+  [bounds_dummy]
+    order = CONSTANT
+    family = MONOMIAL
+  []
+[]
+
+[Kernels]
+  [diff_u]
+    type = Diffusion
+    variable = u
+  []
+  [reaction_u]
+    type = Reaction
+    variable = u
+  []
+
+  [diff_v]
+    type = Diffusion
+    variable = v
+  []
+  [reaction_v]
+    type = Reaction
+    variable = v
+  []
+[]
+
+[DGKernels]
+  [dg_diff_u]
+    type = ADDGDiffusion
+    variable = u
+    epsilon = -1
+    sigma = 6
+    diff = 3
+  []
+  [dg_diff_v]
+    type = ADDGDiffusion
+    variable = v
+    epsilon = -1
+    sigma = 6
+    diff = 4
+  []
+[]
+
+[BCs]
+  [left_u]
+    type = DGFunctionDiffusionDirichletBC
+    variable = u
+    boundary = '0'
+    function = -0.5
+    epsilon = -1
+    sigma = 6
+  []
+
+  [right_u]
+    type = NeumannBC
+    variable = u
+    boundary = 1
+    value = 30
+  []
+
+  [left_v]
+    type = DGFunctionDiffusionDirichletBC
+    variable = v
+    boundary = '0'
+    function = 4
+    epsilon = -1
+    sigma = 6
+  []
+
+  [right_v]
+    type = NeumannBC
+    variable = v
+    boundary = 1
+    value = -40
+  []
+[]
+
+[Bounds]
+  [u_upper_bound]
+    type = ConstantBoundsAux
+    variable = bounds_dummy
+    bounded_variable = u
+    bound_type = upper
+    bound_value = 1
+  []
+  [u_lower_bound]
+    type = ConstantBoundsAux
+    variable = bounds_dummy
+    bounded_variable = u
+    bound_type = lower
+    bound_value = 0
+  []
+
+  [v_upper_bound]
+    type = ConstantBoundsAux
+    variable = bounds_dummy
+    bounded_variable = v
+    bound_type = upper
+    bound_value = 3
+  []
+  [v_lower_bound]
+    type = ConstantBoundsAux
+    variable = bounds_dummy
+    bounded_variable = v
+    bound_type = lower
+    bound_value = -1
+  []
+[]
+
+[Executioner]
+  type = Steady
+
+  solve_type = 'PJFNK'
+[]
```

Comment:
IIRC you have to use `-snes_type vinewtonrsls` or `-snes_type vinewtonssls` for the bounds to be active.

Reply:
Can you also post of a plot of the result in this PR? Thanks.

Reply:
It's passed in the tests spec. It s a good point though I ll move them

Reply:
<img width="655" alt="Screen Shot 2023-02-16 at 3 36 47 PM" src="https://user-images.githubusercontent.com/25037151/219502702-fa5fb9c5-43b2-48dd-b356-6370e19e2afd.png">